uaclient/cli/cli_util.py

```diff
@@ -140,6 +142,24 @@ def new_f(args, cfg, **kwargs):
     return new_f
 
 
+def assert_vulnerability_issue_valid(cmd):
```

Comment:
This saves the repetition of 5 lines by adding 7 more to wrap this in a decorator :/

I'm not sure the complexity is worth it, but no need to change it now. Just my 2cents going forward